webapp/src/components/AssetCard/AssetCard.tsx

```diff
@@ -106,6 +106,7 @@ const AssetCard = (props: Props) => {
       as={Link}
       to={getAssetUrl(asset, isManager && isLand(asset))}
       onClick={onClick}
+      id={`${asset.contractAddress}-${'tokenId' in asset ? asset.tokenId : asset.itemId}`}
```

Comment:
```suggestion
      id={`${asset.contractAddress}-${asset.tokenId ?? asset.itemId}`}
```
would this work? looks a little bit simplier